spec/support/mongoid.rb

```diff
@@ -36,4 +36,15 @@ def have_fields(*args)
 
 RSpec.configure do |configuration|
   configuration.include Mongoid::Matchers
+
+  # rspec-rails 3 will no longer automatically infer an example group's spec type
+  # from the file location. You can explicitly opt-in to the feature using this
+  # config option.
+  # To explicitly tag specs without using automatic inference, set the `:type`
+  # metadata manually:
+  #
+  #     describe ThingsController, :type => :controller do
+  #       # Equivalent to being in spec/controllers
+  #     end
+  configuration.infer_spec_type_from_file_location!
```

Comment:
we already have this configuration set in the [spec helper](https://github.com/igrigorik/vimgolf/blob/master/spec/spec_helper.rb#L28)